recipes/woodwork/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "woodwork" %}
+{% set version = "0.0.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5bee95b42973ed980f5d188372aca864248e5b660bd36fd19bdb9d6d5a72e8a2
+
+build:
+  entry_points:
+    - woodwork = woodwork.__main__:cli
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - numpy >=1.19.1
+    - pandas >=1.1.0
+    - click >=7.1.2
+    - scikit-learn >=0.23.2
+    - python >=3.6.*
+
+test:
+  imports:
+    - woodwork
+    - woodwork.tests
+  requires:
+    - pip >=20.2.2
+    - pytest ==6.0.1
+    - pytest-cov ==2.10.1
+    - pytest-xdist ==2.0.0
+  source_files:
+    - woodwork/tests/*
+    - requirements.txt
+    - test-requirements.txt
+  commands:
+    - pytest woodwork/tests
```

Can we try to add ``pip check`` as well please?
```suggestion
    - pytest woodwork/tests
    - pip check
```